docs/blog/october-contributions-showcase.md

```diff
@@ -0,0 +1,107 @@
+  - name: James Garbutt
+sidebar: false
+date: 2024-11-04
+head:
+  - - meta
+    - property: og:type
+      content: website
+  - - meta
+    - property: og:title
+      content: October contributions showcase
+  - - meta
+    - property: og:url
+      content: https://e18e.dev/blog/october-contributions-showcase
+  - - meta
+    - property: og:description
+      content: A showcase of e18e-focused contributions from October
+---
+
+# September contributions
+
+_October 4, 2024_
+
+A few days late again because I was caught in another typhoon :grimacing:, but here's the latest wins from the e18e community!
+
+# Libraries
+
+## picospinner & nanospinner
+
+Two small but great additions, these CLI spinner libraries are here to replace heavier weight alternatives like `ora`.
+
+[`picospinner`](https://github.com/tinylibs/picospinner/) was recently adopted into the [tinylibs](https://github.com/tinylibs) organisation, and is being lead by [Pondwader](https://github.com/PondWader).
+
+[`nanospinner`](https://github.com/usmanyunusov/nanospinner) author [@usmanyunusov](https://x.com/usmanyunusov) has also recently joined the e18e community and is actively helping move things forward.
+
+Both of these libraries are great alternatives and unblock a whole bunch of future migrations. Looking forward to see more!
+
+## empathic / fd-package-json
+
+As part of the cleanup project, we needed an alternative package for finding the closest `package.json`. To fill the gap, [`fd-package-json`](https://github.com/es-tooling/fd-package-json) was created **and recently passed 800,000 downloads/week!**
+
+However, we still had many gaps where we wanted to traverse for other files and directories without having to pull in a heavy library.
+
+This is where [empathic](https://github.com/lukeed/empathic) comes in!
+
+Another great library by [@lukeed](https://x.com/lukeed05), this provides a bunch of useful utilities for dealing with paths and traversals.
+
+This library can easily replace many of the `find-up`-like packages floating around the ecosystem:
+
+```ts
+import * as find from 'empathic/find'
+
+// Find closest "foobar.config.js" file
+const file = find.up('foobar.config.js', { cwd })
+```
+
+We will soon be migrating many projects to both of these libraries for sure :raised_hands:
+
+# Contributions / Improvements
+
+## Storybook
+
+## Docusaurus
+
+This one didn't originate from e18e but is still well worth a mention. [Docusaurus](https://github.com/facebook/docusaurus) is currently tackling a pretty huge project of [improving build performance](https://github.com/facebook/docusaurus/issues/10556).
+
+Great to see another large project taking a focus on performance :pray:
+
+They have already shaved off minutes from their build times :fire: very impressive work by the team.
+
+# Other news
+
+## `require(esm)` in Node
+
+[@joyeecheung](https://x.com/joyeecheung) and others have recently been working hard on allowing Node to `require(...)` ES modules.
+
```

Comment:
There is also the so-called faux-ESM package (authored in ESM, transpiled to CJS, likely only shipping the CJS version, and could be migrated to ship ESM directly). You can find a list of classified high-impact packages in https://github.com/wooorm/npm-esm-vs-cjs/ (there are some that are labeled incorrectly, e.g. some packages labeled as ESM are actually dual and still could use some cleanup https://github.com/wooorm/npm-esm-vs-cjs/issues/11 , but it's overall a very useful TODO list ;)

Reply:
aha good point. I totally forgot about the high impact repo too, that'll come in useful for tasking out the migrations